galsim/interpolatedimage.py

```diff
@@ -550,6 +546,182 @@ def __setstate__(self, d):
                       _force_stepk=self._stepk, _force_maxk=self._maxk)
 
 
+class InterpolatedKImage(GSObject):
+    """A class describing non-parametric profiles specified by samples -- i.e., images -- of their
+    complex Fourier transform.
+
+    The InterpolatedKImage class is useful if you have a non-parametric description of the Fourier
+    transform of an object as a pair of Images -- one for the real part and one for the imaginary
+    part -- of the complex-valued profile that you wish to manipulate / transform using GSObject
+    methods such as shear(), magnify(), shift(), etc.  Note that neither real-space convolution nor
+    photon-shooting of InterpolatedKImages is currently implemented.  Please submit an issue at
+    http://github.com/GalSim-developers/GalSim/issues if you require either of these use cases.
+
+    The primary intended use case for InterpolatedKImage is to create ChromaticObjects from
+    PSF-deconvolved multi-band HST images, a functionality that will be added in a future
+    pull-request.
+
+    The images required for creating an InterpolatedKImage are precisely those returned by the
+    GSObject `.drawKImage()` method.  The `a` and `b` objects in the following command will produce
+    identical images when drawn with the `.drawImage()` method:
+
+    >>> a = returns_a_GSObject()
+    >>> b = galsim.InterpolatedKImage(*a.drawKImage())
+
+    The real- and imaginary-part Images must have the same data type, same bounds, and same scale.
+    The only wcs permitted is a simple PixelScale.  Furthermore, the complex-valued Fourier profile
+    formed by `real_kimage` and `imag_kimage` must be Hermitian, since it represents a real-valued
+    real-space profile.  The outputs from `drawKImage` automatically meet all of these criteria.
```

Comment:
I'm not sure how useful it is to mention that the output of drawKImage meets these criteria.  It reads to me like you are implying that this is the recommended way for the user to generate these images.  

But presumably if you had an object with which you could call `drawKImage`, then you would not build an InterpolatedKImage with it.  You already have it as a GSObject.  So I'd recommend removing this last sentence.  The previous items are what the user needs to know.